client/galaxy/scripts/mvc/workflow/workflow-forms.js

```diff
@@ -234,9 +234,8 @@ define( [ 'utils/utils', 'mvc/form/form-view', 'mvc/tool/tool-form-base' ], func
                         }
                         if ( input.payload ) {
                             for ( var p_id in input.payload ) {
-                                var p = input.payload[ p_id ];
-                                input.payload[ input.name + '__' + p_id ] = p;
-                                delete p;
+                                input.payload[ input.name + '__' + p_id ] = p;input.payload[ p_id ];
```

Comment:
@guerler I think this may have been broken in a merge, let me look into it.  Thanks!